src/ray/gcs/gcs_server/test/gcs_monitor_server_test.cc

```diff
@@ -43,4 +48,19 @@ TEST_F(GcsMonitorServerTest, TestRayVersion) {
   ASSERT_EQ(reply.version(), kRayVersion);
 }
 
+TEST_F(GcsMonitorServerTest, TestDrainAndKillNode) {
+  rpc::DrainAndKillNodeRequest request;
+  rpc::DrainAndKillNodeReply reply;
+  auto send_reply_callback =
+      [](ray::Status status, std::function<void()> f1, std::function<void()> f2) {};
+
+  *request.add_node_ids() = NodeID::FromRandom().Binary();
+  *request.add_node_ids() = NodeID::FromRandom().Binary();
+
+  EXPECT_CALL(*mock_node_manager_, DrainNode(_)).Times(AtLeast(2));
```

Comment:
Why at least 2, shouldn't it be exactly 2?

Reply:
good point.